tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -155,19 +164,36 @@ def fe_input_user_data_repack(input_model: InputModel, input_user_shapes: [None,
     _input_shapes = []
     if isinstance(input_user_shapes, list) or isinstance(input_user_shapes, dict):
         for input_name in input_user_shapes:
-            node = decode_name_with_port(input_model, input_name, framework)
+            node = decode_name_with_port(
+                input_model, input_name, framework, IOType.Input
+            )
             if node is None:
-                raise Error('Cannot find location {} in the input model'.format(input_name))
-            shape = None if isinstance(input_user_shapes, list) else input_user_shapes[input_name]
+                raise Error(
+                    "Cannot find location {} in the input model".format(input_name)
+                )
+            shape = (
+                None
+                if isinstance(input_user_shapes, list)
+                else input_user_shapes[input_name]
+            )
             if input_user_data_types.get(input_name) is not None:
                 data_type = input_user_data_types[input_name]
-                _input_shapes.append({'node': node, 'shape': shape, 'data_type': data_type, 'input_name': input_name})
+                _input_shapes.append(
+                    {
+                        "node": node,
+                        "shape": shape,
+                        "data_type": data_type,
+                        "input_name": input_name,
+                    }
+                )
             else:
-                _input_shapes.append({'node': node, 'shape': shape, 'input_name': input_name})
+                _input_shapes.append(
+                    {"node": node, "shape": shape, "input_name": input_name}
```

Comment:
keep same notation - for condition above you split list to new lines, here is half-way changed only

Reply:
It is result of using auto formatter `autopep8` on IDE

Reply:
Fixed